README.md

```diff
@@ -1,55 +1,177 @@
 # BcContainerHelper
 
-This repository contains a module, which makes it easier to work with Business Central and NAV Containers on Docker.
-The module is work in progress and feel free to submit pull requests and contribute with good ideas.
-Documentation will be on the wiki.
+BcContainerHelper is a PowerShell module designed to simplify the management of Business Central and Dynamics NAV containers on Docker. As a work-in-progress project, it welcomes contributions through pull requests and ideas. Comprehensive documentation is available in the [NavContainerHelper.md](NavContainerHelper.md).
 
 # Getting Started
 
+There are two primary methods for running these Docker containers: as a [VM on Azure](#vm-on-azure) or [locally](#local-installation).
+
+Using a **VM on Azure** provides a ready-to-go environment within approximately 30 minutes, eliminating the need for manual installation. If you have Azure credits as part of a Visual Studio subscription, you can use them for these VMs, making it a cost-effective option.
+
+While **local installation** requires more initial setup, it offers a fully configured local environment where you can work with Business Central containers even offline.
+
 ## VM on Azure
-Follow the steps on https://aka.ms/getbc
 
-## Local Installation on Windows or Windows Server
-This would be the Steps to install Docker and BCContainerHelper on Windows 11 or Windows Server 2022:
+To set up a VM on Azure, follow the steps provided at [aka.ms/getbc](https://aka.ms/getbc).
+
+## Local Installation
+
+Local installation depends on whether you use [Windows 10 or 11](#local-installation-on-windows-10-or-11) or [Windows Server](#local-installation-on-windows-server).
+
+You can choose between Hyper-V isolation or Process isolation:
+
+| Feature                        | Hyper-V Isolation                                                      | Process Isolation                                             |
+|--------------------------------|------------------------------------------------------------------------|----------------------------------------------------------------|
+| **Isolation Level**            | High: Uses a separate hypervisor-based virtual machine                 | Moderate: Uses namespace and resource control to isolate containers within the same OS instance |
+| **Performance**                | Lower: Due to the overhead of running a full VM                        | Higher: Less overhead, as it runs directly on the host OS kernel  |
```

Comment:
On my Windows 11 machine - HyperV out-performs Process isolation by factor 4 - yes, 4 times faster.
I don't know why, but I assume that antivirus and other services are causing this.
I also have a partner who, when running windows server (same host and container version) are experiencing random issues running tests when using process isolation - problems, which goes away when enabling HyperV isolation.

So, I am not sure it is as black and white as this.

I will merge the PR and make some adjustments based on my experiences afterwards.